### ISSUE TITLE
How do you prevent model overfit?

### ISSUE THREAD

**Issue by raikel** [WaqasSultani/AnomalyDetectionCVPR2018#67]
I don't see any validation during training. Training could easily go into overfitting is model is not checked against a validation set.
